elixir/wrapper/module.py

```diff
@@ -5,6 +5,7 @@
 
 import torch
 import torch.nn as nn
+from colossalai.utils.model.experimental import LazyTensor
```

Comment:
@ver217 
Could you use the function `_lazy_init_check(param)` to check lazy init?

Reply:
done

Reply:
...There is one function in the module which is prepared for you.

Reply:
updated